smogv2.py

```diff
@@ -108,8 +119,30 @@ def perlin_noise(world, scale=600.0, octaves=6, persistence=0.5, lacunarity=2.0)
 
 
 if __name__ == "__main__":
-    path_to_json = sys.argv[1]
-    save_dir = sys.argv[2]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--json", "-j", type=str)
+    parser.add_argument("--save_dir", "-s", type=str, default="")
+    parser.add_argument("--comet", "-c", action="store_true", default=False)
+    parser.add_argument("--no_pert", action="store_true", default=False)
```

Comment:
The name of this param is confusing. if it is no_pert = False then it means there is perturbation 🤔  ... maybe if that's what you are thinking, you might want to set the default to no_pert = True because results look better without perlin noise